esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -856,15 +856,17 @@ void EspHttpBinding::handleHttpPost(CHttpRequest *request, CHttpResponse *respon
     StringBuffer cacheID;
     unsigned cacheSeconds = 0;
     IEspCache *cacheClient = nullptr;
+    IEspContext &context = *request->queryContext();
 
     IEspContainer *espContainer = getESPContainer();
     if (espContainer->hasCacheClient() && (cacheMethods > 0)
         && queryCacheSeconds(request->queryServiceMethod(), cacheSeconds)) //ESP cache is needed for this method
     {
-        cacheClient = (IEspCache*) espContainer->queryCacheClient(getCacheGroupID());
+        cacheClient = (IEspCache*) espContainer->queryCacheClient(getCacheGroupID(request->queryServiceMethod()));
         if (cacheClient)
             createESPCacheID(request, cacheID);
-        if (!cacheID.isEmpty() && sendFromESPCache(cacheClient, request, response, cacheID.str()))
+        if (!cacheID.isEmpty() && !context.queryRequestParameters()->queryProp("not_read_esp_cache")
```

Comment:
Since you are still going to write to the cache, but just not just return the existing cache value I think "not_read" is a little confusing.  You are 1. bypassing the cache, but then 2. updating the cache.  I would say "update_cache" would basically describe the behavior,.. although not if added to a method without cache at all.

"dirty_cache" is another option... which means exactly "the saved cache is no good, update it and return the new value".

Reply:
Will change to "dirty_cache".